config/cscs.py

```diff
@@ -4,12 +4,12 @@
 
 
 class ReframeSettings:
-    _reframe_module = 'reframe'
-    _job_poll_intervals = [1, 2, 3]
-    _job_submit_timeout = 60
-    _checks_path = ['checks/']
-    _checks_path_recurse = True
-    _site_configuration = {
+    reframe_module = 'reframe'
+    job_poll_intervals = [1, 2, 3]
+    job_submit_timeout = 60
+    checks_path = ['checks/']
+    checks_path_recurse = True
+    site_configuration = {
```

Comment:
I made all these attributes public and removed properties. See the PR description for a rationale. Opinions?